src/control/cmd/dmg/storage.go

```diff
@@ -116,54 +119,52 @@ func (cmd *storageScanCmd) Execute(_ []string) error {
 		return errors.New("Cannot use --nvme-health and --nvme-meta together")
 	}
 
-	ctx := context.Background()
 	req := &control.StorageScanReq{NvmeHealth: cmd.NvmeHealth, NvmeMeta: cmd.NvmeMeta}
 	req.SetHostList(cmd.hostlist)
-	resp, err := control.StorageScan(ctx, cmd.ctlInvoker, req)
+	resp, err := control.StorageScan(context.Background(), cmd.ctlInvoker, req)
+	if err != nil {
+		return err
+	}
 
 	if cmd.jsonOutputEnabled() {
 		if cmd.Verbose {
 			cmd.log.Error("--verbose flag ignored if --json specified")
 		}
```

Comment:
I vaguely remember pointing this out before, but I think this should be removed, as it's kind of a pointless message. It shouldn't break JSON output because it will appear on stderr, but I don't think it helps anyone.